### PR TITLE
Documentation update for Android and iOS

### PR DESCRIPTION
**Pull request by umangmoe**
### Proposed changes
Updated APIs for MoEngage Android and iOS integration.

### Merge timing
- On a specific date? 30th March 2022

### CODE REVIEW

File: src/connections/destinations/catalog/moengage/index.md [hunk 1]
```diff
@@ -27,30 +27,41 @@ users it’ll be instantaneous! Segment-MoEngage Integration is a bundled integr
...
-  pod 'Segment-MoEngage’
+  pod 'Segment-MoEngage'
   ```
 
    * Update the pod.
 
     pod update
 
+
+ To install with SPM use the https://github.com/moengage/MoEngage-Segment-iOS.git and set the branch as master or version as 7.0.0 and above.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/moengage/index.md:45]:
```suggestion
 To install with SPM use the [MoEngage-Segment-iOS](https://github.com/moengage/MoEngage-Segment-iOS.git){:target="_blank"} library and set the branch as master or version as 7.0.0 and above.
```

File: src/connections/destinations/catalog/moengage/index.md [hunk 2]
```diff
@@ -139,19 +150,19 @@ Push Notifications are a great way to keep your users engaged and informed about
...
 
 
 #### Cards
-Create targeted or automated App Inbox/NewsFeed messages that can be grouped into various categories, and target your users with different updates or offers that can stay in the Inbox/Feed over a designated period of time. For more information about cards, see [MoEngage - Cards](https://docs.moengage.com/docs/ios-cards).
+Create targeted or automated App Inbox/NewsFeed messages that can be grouped into various categories, and target your users with different updates or offers that can stay in the Inbox/Feed over a designated period of time. For more information about cards, see [MoEngage - Cards](https://developers.moengage.com/hc/en-us/articles/4404058438676-Cards-in-iOS){:target="_blank"}.
 
 
 #### Compliance
-To make the App compliant with policies i.e,GDPR while using our SDK, follow the doc in this [link](https://docs.moengage.com/docs/gdpr-compliance-1).
+To make the App compliant with policies i.e,GDPR while using our SDK, follow the doc in this [link](https://developers.moengage.com/hc/en-us/articles/4403905438228-SDK-initialisation){:target="_blank"}.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/moengage/index.md:165]:
```suggestion
To make the App compliant with policies (such as GDPR) while using MoEngage's SDK, follow the instructions in this [doc](https://developers.moengage.com/hc/en-us/articles/4403905438228-SDK-initialisation){:target="_blank"}.
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
Thanks for the contribution @umangmoe! Just suggested a few small changes. 
As requested, I can merge this 3/30 and it will be deployed to our docs site the next day.

File: src/connections/destinations/catalog/moengage/index.md (hunk 1 above)
Comment (umangmoe) [reply src/connections/destinations/catalog/moengage/index.md:45]: updated.

File: src/connections/destinations/catalog/moengage/index.md (hunk 2 above)
Comment (umangmoe) [reply src/connections/destinations/catalog/moengage/index.md:165]: Updated.

**Review by rchinn-segment — APPROVED** [review]

**Pull request merged**